src/grpc/dp_grpc_impl.c

```diff
@@ -25,6 +25,14 @@
 
 static uint32_t pfx_counter = 0;
 
+static __rte_always_inline bool dp_is_ipv6_addr_zero(const uint8_t *addr)
+{
+	for (int i = 0; i < DP_IPV6_ADDR_SIZE; i++)
+		if (addr[i] != 0)
+			return false;
+	return true;
+}
```

Comment:
May be over-optimizing, since I do not know how often this is called, but maybe for the future it would be better to optimize anyway?
```
assert(DP_IPV6_ADDR_SIZE == 2*sizeof(uint64_t), "something something addr size");
return *((const uint64_t *)addr) == 0 && *((const uint64_t *)(addr+sizeof(uint64_t))) == 0;
```

Reply:
Sounds good to me.